src/documentation/index.md

```diff
@@ -40,9 +40,9 @@ A Beam Runner runs a Beam pipeline on a specific (often distributed) data proces
 * [DirectRunner]({{ site.baseurl }}/documentation/runners/direct/): Runs locally on your machine -- great for developing, testing, and debugging.
 * [ApexRunner]({{ site.baseurl }}/documentation/runners/apex/): Runs on [Apache Apex](http://apex.apache.org).
 * [FlinkRunner]({{ site.baseurl }}/documentation/runners/flink/): Runs on [Apache Flink](http://flink.apache.org).
+* [GearpumpRunner]({{ site.baseurl }}/documentation/runners/gearpump/): Runs on [Apache Gearpump (incubating)](http://gearpump.apache.org).
```

Comment:
I would actually like to have this content on the site, even while we leave it as "under development". This will get it a little bit more publicity for anyone interested.

How about this:

 - Don't do this change to `documentation/index`
 - The content in `documentation/runners/gearpump/index` could stay there but add a little bit about building from the feature branch
 - Don't link it from the `header`
 - _Do_ link to it from `contribute/work-in-progress/`, replacing what is currently a link to the Gearpump runner `README`.